conans/test/integration/command/remote_test.py

```diff
@@ -278,3 +278,16 @@ def test_invalid_url(self):
                       self.client.out)
         self.client.run("remote list")
         self.assertIn("pepe.org", self.client.out)
+
+
+def test_duplicated_url():
+    """ allow duplicated URL with --force
```

Comment:
@memsharded this comment implies `--force` would be needed for this to work, but I do not see such either in the commands below or the related code. Is this a bug or a typo?

Reply:
You are right, the comment is obsolete. No --force is necessary, it was replaced with a warning. Do you think that we should change it to ``--force``?

Reply:
That would certainly be my expectation for this, yes :)

Reply:
Sure, yes, I agree, lets change it

Reply:
https://github.com/conan-io/conan/pull/13112